packages/react-vapor/src/components/editor/CodeEditor.tsx

```diff
@@ -59,6 +60,11 @@ export class CodeEditor extends React.Component<ICodeEditorProps, CodeEditorStat
 
     componentDidMount() {
         this.props.onMount?.(this.codemirror.current);
+        this.props.isRefreshRequired && setInterval(this.timer, 300);

```

Comment:
Do we really want to refresh the editor every 0,3 s ? Maybe a setTimeout would be more appropriate ?

Reply:
The setTimeout wouldn't work in that case, because the editor is mounted about the same time as the CollapsibleContainer component. So, if you don't open the collapsible before the time is up, the editor's content won't be shown. However, I do agree it's not an ideal solution. It should trigger a setTimeout only when the collapsible is opened.